featuretools/tests/synthesis/test_deep_feature_synthesis.py

```diff
@@ -448,6 +449,35 @@ def test_make_rolling_count_off_datetime_feature(pd_es):
     assert feature_with_name(features, rolling_transform_name)
 
 
+def test_numeric_lag_works_with_non_nullable(pd_es):
```

Comment:
Just curious about the intended purpose of this test. Were you intending to make sure we can calculate feature values correctly and that we don't get errors if we have a non-nullable input that gets lagged and introduces null values?

If so, I think you actually need to go to the step of computing the feature matrix instead of just the features. Without computing the features, I don't think that type of error will show up.

Feel free to disregard if the intention was something else here.

Reply:
yep, you're right. I can add the calculate feature matrix step here. Though maybe that means this should be a more targeted test in `test_calculate_feature_matrix` or `test_feature_set_calculator`

Reply:
Yeah, probably would be best to relocate the test in that case.